dist/rpm/os-autoinst-scripts-deps.spec

```diff
@@ -41,19 +43,23 @@ Suggests:       postgresql
 %package devel
 Summary:        Development package for os-autoinst-scripts-deps
 Group:          Development/Tools/Other
+BuildRequires:  %devel_requires
 Requires:       %devel_requires
 
 %description devel
 Development package pulling in all dependencies needed for developing
 in os-autoinst-scripts-deps
 
 %prep
+%setup -n scripts-%{version}
 
 %build
 
 %install
 
 %check
+# Can't run bash tests because it needs to clone bpan via git
+make test-python
```

Comment:
I wonder if just `%pytest` would do here? as described in https://en.opensuse.org/openSUSE:Packaging_Python#Running_tests